dace/frontend/fortran/fortran_parser.py

```diff
@@ -3010,30 +3010,14 @@ def create_sdfg_from_string(
         raise NameError("Not all functions were transformed to subroutines")
     program.function_definitions = []
     program = ast_transforms.SignToIf().visit(program)
-    program = ast_transforms.ArgumentExtractor(program).visit(program)
-
-    program = ast_transforms.TypeInference(program, assert_voids=False).visit(program)
+    program = ast_transforms.ArrayToLoop(program).visit(program)
```

Comment:
Is it necessary to revert that? IIRC ecrad parser shouldn't even use that call inside fortran_parser, so this can stay. It only affects tests

Reply:
Undid this part in a new commit.